plugins/hypervisors/kvm/src/main/java/com/cloud/hypervisor/kvm/storage/KVMStoragePool.java

```diff
@@ -92,6 +92,8 @@ public default KVMPhysicalDisk createPhysicalDisk(String volumeUuid, PhysicalDis
 
     public Map<String, String> getDetails();
 
+    String getLocalPathFor(String relativePath);
```

Comment:
```suggestion
    default String getLocalPathFor(String relativePath) {
        return String.format("%s%s%s", getLocalPath(), File.separator, relativePath);
    }
```

add default implementation , and remove overrides with similar implementation